Formula/lib/libmsquic.rb

```diff
@@ -0,0 +1,45 @@
+class Libmsquic < Formula
+  desc "Cross-platform, C implementation of the IETF QUIC protocol"
+  homepage "https://github.com/microsoft/msquic"
+  url "https://github.com/microsoft/msquic.git",
+  tag:      "v2.4.4",
+  revision: "e87ae9a37bcebdf5ab3725e5d882ddb50e4dbbb4"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "openssl@3"
+
+  def install
+    system "cmake", "-B", "build/openssl3",
+    "-DQUIC_OUTPUT_DIR=../../../artifacts/bin/Release_openssl3",
+    "-DCMAKE_BUILD_TYPE=Release", "-DQUIC_TLS=openssl3",
+    "-DQUIC_USE_SYSTEM_LIBCRYPTO=true", "-DQUIC_BUILD_PERF=off",
+    "-DQUIC_BUILD_TOOLS=off"
+    system "cmake", "--build", "build/openssl3"
+    system "cmake", "--install", "build/openssl3", "--prefix", prefix
```

Comment:
Let's follow the style used by other formulae.
```suggestion
    system "cmake", "-S", ".", "-B", "build/openssl3",
                    "-DQUIC_OUTPUT_DIR=../../../artifacts/bin/Release_openssl3",
                    "-DQUIC_TLS=openssl3",
                    "-DQUIC_USE_SYSTEM_LIBCRYPTO=true",
                    "-DQUIC_BUILD_PERF=off",
                    "-DQUIC_BUILD_TOOLS=off",
                    *std_cmake_args
    system "cmake", "--build", "build/openssl3"
    system "cmake", "--install", "build/openssl3"
```
The value of `QUIC_OUTPUT_DIR` looks suspicious.

Reply:
removing that in a bit, forgot to remove it.